packages/store/src/models/store.ts

```diff
@@ -35,6 +35,9 @@ export default class Store {
   @Property({ columnType: "text", default: "Medusa Store" })
   name: string
 
+  @Property({ type: "array", default: "{}" })
```

Comment:
Suggestion: i would default to [] as it is a valid jsonb value

Reply:
@adrien2p why do we want it as jsonb, where Postgres has native support for primitive type arrays (and IIRC "{}" is how you represent an empty array, but maybe mikroorm does some magic if you pass []?)

Reply:
My bad, mostly a habit, also jsonb areay are declare like this [] and with array you can either use {} or ARRAY[]. Feel free to discard

Reply:
@adrien2p No worries, I am always wondering if there is some magic happening that I should know of 😄